examples/print_affinity.cpp

```diff
@@ -53,22 +53,22 @@ int main(int argc, char** argv)
     for (int i = 0; i < omp_get_num_threads(); ++i) {
         int cpu = sched_getcpu();
 #pragma omp ordered
-{
-        if (i) {
-            thread_list << ",";
+        {
+            if (i) {
+                thread_list << ",";
+            }
+            thread_list << cpu;
         }
-        thread_list << cpu;
-}
     }
 #endif
 
     for (int rank = 0; rank < world_size; ++rank) {
         if (rank == world_rank) {
             // Print off affinity message
-            std::cout << "Rank: " << world_rank
-                      << " cgroup CPUs: [" << cpu_list.str() << "]"
-                      << " omp CPUs: [" << thread_list.str() << "]"
-                      << std::endl << std::flush;
+            std::cout << "Rank: " << world_rank << " cgroup CPUs: ["
+                      << cpu_list.str() << "]"
+                      << " omp CPUs: [" << thread_list.str() << "]" << std::endl
+                      << std::flush;
```

Comment:
I liked it better the way it was.  This makes it difficult to see what's being printed a bit.  Maintenance issue?

Reply:
eh, it isn't that easy to read either way